app/services/backup_service.rb

```diff
@@ -143,6 +143,7 @@ def download_to_tar(tar, attachment, filename)
       end
     end
   rescue Errno::ENOENT
```

Comment:
This should be: `rescue Errno::ENOENT, Seahorse::Client::NetworkingError`

Reply:
Done